fab/environments.yml

```diff
@@ -110,19 +110,15 @@ enikshay:
       GroupToUserPillow:
         num_processes: 1
       kafka-ucr-main:
-        num_processes: 4
+        num_processes: 2
       KafkaDomainPillow:
         num_processes: 1
       LedgerToElasticsearchPillow:
         num_processes: 1
-      ReportXFormToElasticsearchPillow:
```

Comment:
These aren't needed. I've been meaning to look into seeing if we can just trash them. It's some custom thing that's been hard to track down